proposals/mftf-package.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+Magento Functional Testing Framework (MFTF) with 2.3.0 release is going to be a require-dev dependency in main magento composer.json because test location was changed from `dev/tests/acceptance` to `app/code/<vendor_name>/<module_name>/Test/Mftf`.
+This leads us to open question where to keep Framework packages.
+
+MFTF have two options:
+1. repo.magento.com
+2. packagist.org
+
+# repo.magento.com
+Is the Composer package repository which owned by Magento.
+
+### Benefits
+1. Magento is fully capable to control released packages.
+2. No need to contact to 3rd party vendors in case of theirs downtime.
+3. Consistency. All magento packages and extensions published in repo.magento.com.
+
+### Disadvantages
+1. Basic Magento Installation from GitHub will require adding credentials to composer auth.json.
+2. Possible additional DevOps work needed (creating a build) to publish packages.
+
+# packagist.org
+Is the default Composer package repository which already handles several Magento packages:
+https://packagist.org/users/mage2-team/
+
+### Benefits
+1. No need to update composer auth.json because packagist is the default for composer.
+2. No DevOps work needed to publish package, except adding MFTF GitHub repo to packagist.org.
+
+### Disadvantages
+1. Inconsistency. Most Magento packages are not located here.
+
+# Summary
+On my opinion packagist.org is the right choice for MFTF because it is not a production component which means Magento customers will not be affected at all.
+And because developer experience will not be affected by adding creds to auth.json. 
```

Comment:
Currently we plan to publish only Framework to packagist.org
Test packages we plan to publish to repo.magento.com
Also we thinking about having only Framework in both composer repos, to support github installation and composer installation of Magento Testing Framework

Reply:
I think it would be good not to make users who want to install Magento Open Source modify auth.json.

There are the following solutions
1. Publish framework on packagist. Maybe in the future we can treat packagist as repository for publicly available packages and repo.magento.com for commercially available packages? I think this would be more logical, but more work for us potentially.
2. Do not add dependency on MFTF framework to root composer.json. Developers who need MFTF will have to install it manually.
3. Use git repository.

@okolesnyk what do you think about option 3? I think it should not require authorization since it's public repo.

Reply:
@melnikovi I don't like idea that Magento composer.json will have only one reference to git repo because of MFTF.
And I don't like this because we already have Magento/ZF1 on packagist.
But thank you for a suggestions.